status-im/status-react
# chore: update to new ui on intro screen of syncing

fixes: https://github.com/status-im/status-mobile/issues/15448

Implements the new ui for the first page of the syncing flow in app.

Notes: 
This pr does not change any logic, only ui.
There will be follow up prs/issues to take on the next screens.

These flows are only in this theme.

To test:
Go to Profile -> Syncing and see that screen.

<img width="307" alt="Screenshot 2023-03-22 at 15 43 18" src="https://user-images.githubusercontent.com/22799766/226959078-cb12ba48-ae9e-47e6-985b-bd1cb10c0196.png">
